### PR TITLE
mes-passCertificateAnalytics

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
Added analytics to pass-certificates
## Checklist

- [ ] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/pass-certificates/pass-certificates.actions.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { createAction } from '@ngrx/store';
+
+export const PassCertificateDidEnter = createAction(
```
Comment (matthew2564) [inline src/app/pages/pass-certificates/pass-certificates.actions.ts]: Update to - `PassCertificatesViewDidEnter`

File: src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+@Injectable()
+export class PassCertificatesAnalyticsEffects {
+
+  constructor(
+    private analytics: AnalyticsProvider,
+    private actions$: Actions,
+  ) {
+  }
+
+  journalView$ = createEffect(() => this.actions$.pipe(
```
Comment (matthew2564) [inline src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts]: Update to `passCertificatesView$`

File: src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    ofType(PassCertificateDidEnter),
+    switchMap(() => {
+      this.analytics.setCurrentPage(AnalyticsScreenNames.PASS_CERTIFICATES);
+      this.analytics.addCustomDimension(AnalyticsDimensionIndices.CANDIDATE_ID, '');
+      this.analytics.addCustomDimension(AnalyticsDimensionIndices.APPLICATION_REFERENCE, '');
```
Comment (matthew2564) [inline src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts]: There will be no Candidate ID or App ref as this is outside the scope of a test.

All that needs to be there is the line that calls `setCurrentPage`

File: src/app/pages/pass-certificates/pass-certificates.actions.ts (hunk 1 above)
Comment (RLCorp) [reply src/app/pages/pass-certificates/pass-certificates.actions.ts]: fixed in new commit

File: src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts (hunk 3 above)
Comment (RLCorp) [reply src/app/pages/pass-certificates/pass-certificates.analytics.effects.ts]: fixed in new commit

**Review by matthew2564 — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]
